firmware/src/core/screenmanager/ScreenManager.cpp

```diff
@@ -24,7 +24,7 @@ ScreenManager::ScreenManager(TFT_eSPI &tft) : m_tft(tft) {
 
     // I'm not sure which cache size is actually good.
     // Needs testing.
-    m_render.setCacheSize(128, 128, 8192);
+    m_render.setCacheSize(8, 8, 4096);
```

Comment:
I really don't know. I mean, it's a cache, so it speeds things up and eats memory.
But I'll add another comment.

Reply:
I just reduced the cache size because I was getting out of memory problems. And the speed was still ok.
If I comment out the line (and hence set the cache size to minimum), the performance impact is pretty big.